src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+const sortTable = function(index, type) {
+  const tbody = table.querySelector('tbody');
+
+  const compare = function(rowA, rowB) {
+    const rowADate = rowA.cells[index].innerHTML;
+    const rowBDate = rowB.cells[index].innerHTML;
+
+    switch (type) {
+      case 'Name':
+      case 'Position':
+        if (rowADate < rowBDate) {
+          return -1;
+        } else if (rowADate > rowBDate) {
+          return 1;
+        }
+        return 0;
+      case 'Age':
+        return rowADate - rowBDate;
+      case 'Salary':
+        const dateA = rowADate.split('$').join('').split(',').join('');
```

Comment:
it would be nice to create a function for getting data to not duplicate this calculation, for example `getSalaryNumber()`